### PR TITLE
New version: Bitnami.ReportServerCommunity version 4.2.0.6066-0

### PR DESCRIPTION
**Pull request by russellbanks**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66324)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by russellbanks** [comment]
I'm not sure how to fix the https://github.com/microsoft/winget-pkgs/labels/Validation-Unattended-Failed

File: manifests/b/Bitnami/ReportServerCommunity/4.2.0.6066-0/Bitnami.ReportServerCommunity.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  Silent: --mode unattended --unattendedmodeui none
+  SilentWithProgress: --mode unattended --unattendedmodeui minimalWithDialogs
+  Interactive: --mode qt
+  InstallLocation: --prefix <INSTALLPATH>
+  Log: --debugtrace <LOGPATH>
+  Custom: --launch_cloud 0
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.bitnami.com/files/stacks/reportserver/4.2.0.6066-0/bitnami-reportserver-4.2.0.6066-0-windows-x64-installer.exe
+  InstallerSha256: 8967933FD30AC21B543044BE75FE14E5004A985AEBF73496F63768DB795C2CC0
```
Comment (ItzLevvie) [inline manifests/b/Bitnami/ReportServerCommunity/4.2.0.6066-0/Bitnami.ReportServerCommunity.installer.yaml:18]:
```suggestion
  InstallerSha256: 8967933FD30AC21B543044BE75FE14E5004A985AEBF73496F63768DB795C2CC0
  AppsAndFeaturesEntries:
  - DisplayName: ReportServer Community packaged by Bitnami
    Publisher: bitnami
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**